prac_06/guitars.py

```diff
@@ -0,0 +1,33 @@
+"""Create list of user's guitars."""
+
+from guitar import Guitar
+
+
+def main():
+    """Displays list of user's guitars."""
+    guitars = []
+
+    print("My Guitars!")
+    name = input("Name: ")
+    while name != "":
+        year_created = input("Year: ")
+        cost = input("Cost: ")
+        guitar = Guitar(name, year_created, cost)
+        guitars.append(guitar)
+        print("{guitar} added.".format(guitar=guitar))
+        print()
```

Comment:
This line may not be necessary as it doesn't print anything.

Reply:
Just for formatting to look like the output shown in the README.md.